src/ingestion/utils.ts

```diff
@@ -154,3 +155,10 @@ export function chainToElements(chain: string): Element[] {
 
     return elements
 }
+
+export function timeoutGuard(message: string): NodeJS.Timeout {
+    return setTimeout(() => {
+        console.log(`⌛⌛⌛ ${message}`)
+        Sentry.captureMessage(message)
+    }, 30000)
+}
```

Comment:
This is only a notification, not an exit from the function timing out, is that correct?

Reply:
yeah, for now just so that we see anything